### PR TITLE
PR: Add option to toggle code style checks from Source menu

### LINKED ISSUE #5488: Code style (pep8) toggle in source menu
## Issue

I often find myself switching the code style warnings on as off as I switch between files and I think it's about 6 clicks through the preferences to get at it.

I often write scripts for figures or data crunching that are one offs and the pep8 is just noise.

For code that I or some one might use again code style is nice to turn on.  Even at that I usually only turn it on in the clean up stage. 

## Solution?

Add a `Code Style Warnings (PEP8)` toggle in the source menu.

Edit: I just found the `# analysis:ignore ` comment.  A `# analysis:ignore_all ` might be handy too.

### PR DESCRIPTION
**Pull request by bcolsen**
Fixes #5488

A pull request that fixes issue #5488

### CODE REVIEW

File: spyder/plugins/editor.py [hunk 1]
```diff
@@ -1020,11 +1020,16 @@ def get_plugin_actions(self):
                 _("Show selector for classes and functions."),
                 'show_class_func_dropdown', 'set_classfunc_dropdown_visible')
 
+        showcode_analysis_pep8_action = self._create_checkable_action(
+                _("Show code syle warnings (pep8)"),
```
Comment (rlaverde) [inline spyder/plugins/editor.py]: *style

Comment (bcolsen) [reply spyder/plugins/editor.py]: oops.  Fixed it.

**Comment by bcolsen** [comment]
It should be good now.  I'm finding it handy already.

**Comment by ccordoba12** [comment]
Thanks for taking the time to help us with this one @bcolsen! I'll merge it as soon as everything is green ;-)

**Pull request merged**